front/.eslintrc.js

```diff
@@ -33,6 +34,12 @@ module.exports = {
     {
       files: ['*.js', '*.jsx', '*.ts', '*.tsx'],
       rules: {
+        'react/no-unescaped-entities': 'off',
```

Comment:
disabling these as the new plugin is stricter. I think it would be great to reactivate jsx-key one, others do not seem that useful (I'm not sure about prop-types)

Reply:
Alright